guidance/02-quality-assured/pages/06-peer-reviewed.qmd

```diff
@@ -1,4 +1,87 @@
 ---
 title: "Peer-reviewed"
 description: ""
----
\ No newline at end of file
+---
+
+> ### Pre-requisite reading
+> * [AF Duck Book: Peer review][duck-book]
+> * [NHSBSA DDaT Playbook: Development - Peer Review][ddat-playbook]
+
+## What is peer review?
+Peer review is a process where a second person will review a colleagues code before any code is finalised or used to produce output.
+
+## Why should I do this?
+Although peer reviews can reduce potential bugs being missed, the peer review process is not simply a case of trying to catch errors.
+
+There are a number of benefits that to using a peer review approach, including:
+
+* **Improving quality of code**: As part of the peer review process code will be reviewed to ensure that it is readable and understandable
+* **Sharing knowledge**: Review code can improve the knowledge of both the original code author and the reviewer
+* **Optimising code performance**: As part of the review process, different approaches may be found that could improve the code
+* **Find potential mistakes in the code**: Although peer review is not a replacement for testing, it can find potential bugs or issues in the code
```

Comment:
The style of bullet-points is not consistent across (maybe also within) pages. Best addressed at a later stage, so will add issue for task of deciding on style and making everything consistent.

Reply:
A couple of further benefits:

* **Consistency**: Promotes consistency by ensuring adherence to team standards and style guides
* **Alternative Perspectives**: Provides alternative perspectives that might lead to simpler or more robust solutions